### ISSUE TITLE
Possible typing error in '--is-isolated-domain' description

### ISSUE THREAD

**Issue by Nuh-prime** [SharePoint/sp-dev-docs#7163]
A possible minor error in instructions for Command-line options for the Yeoman SharePoint generator. In the last command 
`--is-domain-isolated`, the description given is `If 'y', web part will be rendered in isolated domain using IFrame. If set to "y", sets the component type as web part.` 

Both if conditions refer to the same case 'y', in single and double quotes. Perhaps, the latter case should be 'n'.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f2345258-f135-178a-a5af-9705ebafa081
* Version Independent ID: 90ee54a0-1409-6f4d-5018-c9f3e26e46ef
* Content: [Scaffold projects by using Yeoman SharePoint generator](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/toolchain/scaffolding-projects-using-yeoman-sharepoint-generator)
* Content Source: [docs/spfx/toolchain/scaffolding-projects-using-yeoman-sharepoint-generator.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/toolchain/scaffolding-projects-using-yeoman-sharepoint-generator.md)
* Product: **sharepoint**
* Technology: **sharepoint-framework**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by KevinTCoughlin** [comment]
@Nuh-prime it looks like the documentation table is out-of-date. This is the current output of the generator options

```shell
❯ yo @microsoft/sharepoint --help
Usage:
  yo @microsoft/sharepoint:app [options]

Generate a SharePoint client-side solution with webparts.

Options:
  -h,   --help                     # Print the generator's options and usage
        --skip-cache               # Do not remember prompt answers
                                                                                            Default: false
        --skip-install             # Do not automatically install dependencies
                                                                                            Default: false
        --component-type           # The type of component. Currently "webpart", "extension", "library", or "adaptiveCardExtension" (beta)
        --component-description    # Web part description
        --component-name           # Web part name
        --framework                # Framework to use. Available: "none", "react"
        --plusbeta                 # Use the beta packages
        --extension-type           # The type of extension. Currently "ApplicationCustomizer", "FieldCustomizer", "ListViewCommandSet", or "SearchQueryModifier" (beta)
        --solution-name            # Solution name, as well as folder name
        --environment              # The target environment for the solution. "spo".
        --package-manager          # The package manager for the solution. Options are: "npm", "pnpm", or "yarn"
        --skip-feature-deployment  # If specified, allow the tenant admin the choice of being able to deploy the components to all sites immediately without running any feature deployment or adding apps in sites
        --is-domain-isolated       # If specified, the solution will have permissions to access web APIs that are unique not shared with other components in the tenant
```

**Comment by andrewconnell** [comment]
Fixed in PR #7189
